src/olympia/migrations/1145-db-schema-appversions.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `appversions`
+    MODIFY `id` int(10) unsigned NOT NULL,
```

Comment:
Got `Cannot change column 'id': used in a foreign key constraint 'applications_versions_max_6e57db5a_fk_appversions_id' of table 'olympia_dev_copy.applications_versions`